src/test/mir-opt/uninhabited_enum_branching.main.SimplifyCfg-after-uninhabited-enum-branching.after.mir

```diff
@@ -50,7 +50,7 @@ fn main() -> () {
         _6 = const "D";                  // scope 0 at $DIR/uninhabited_enum_branching.rs:27:21: 27:24
                                          // mir::Constant
                                          // + span: $DIR/uninhabited_enum_branching.rs:27:21: 27:24
-                                         // + literal: Const { ty: &str, val: Value(Slice { data: Allocation { bytes: [68], relocations: Relocations(SortedMap { data: [] }), init_mask: InitMask { blocks: [1], len: Size { raw: 1 } }, align: Align { pow2: 0 }, mutability: Not, extra: () }, start: 0, end: 1 }) }
+                                         // + literal: Const { ty: &str, val: Value(Slice { data: Allocation { bytes: [68], relocations: Relocations(SortedMap { data: [] }), init_mask: InitMask { blocks: [1], len: Size::from_bytes(1) }, align: Align::from_bytes(1), mutability: Not, extra: () }, start: 0, end: 1 }) }
```

Comment:
Wow this looks terrible... I think we are better off just not printing all those details.^^

Reply:
In mir dumps, definitely, I'd be wary about the regular debug impls tho, as we want all that info in case we print it in tracing statements I'd think

Reply:
Yeah that's why in my last commit I changed MIR dumps, not the debug impls.